server/src/main/java/org/elasticsearch/search/internal/ShardSearchRequest.java

```diff
@@ -196,11 +196,18 @@ public static long computeWaitForCheckpoint(Map<String, long[]> indexToWaitForCh
 
     // Used by ValidateQueryAction, ExplainAction, FieldCaps, TermsEnumAction, lookup join in ESQL
```

Comment:
As noted, additional handling may be needed for lookup join.

Reply:
Currently, resizing lookup indices is not allowed.